generators/server/templates/src/test/java/package/web/rest/_UserResourceIntTest.java

```diff
@@ -192,12 +192,24 @@ public static User createEntity(<% if (databaseType === 'sql') { %>EntityManager
         return user;
     }
 
+    <%_ if (databaseType === 'sql') { _%>
+    /**
+     * When createEntity is called from other classes is mandatory to create different
+     * login and mail since those have a unique constraint.
+     */
+    private static User createEntityWithDefaults(EntityManager em){
```

Comment:
Is this method really needed ? It could be inlined in the init() method.

Reply:
Sure thing 👍 it just happen 😉 . The method was having for me more of a documentation role, but indeed trough that the code was to verbose, so I just removed.